sdk/src/main/scala/com/horizen/api/http/SidechainTransactionApiRoute.scala

```diff
@@ -288,12 +301,14 @@ case class SidechainTransactionApiRoute(override val settings: RESTApiSettings,
   def sendCoinsToAddress: Route = (post & path("sendCoinsToAddress")) {
```

Comment:
As I understood, it is a method for simple sending coins inside sidechain.
Let's leave it as is (without withdrawals, only regular box outputs) and create another method `withdrawcoins` with withdrawals only.